src/main/java/programminglife/model/drawing/DrawableSegment.java

```diff
@@ -212,6 +228,11 @@ public int hashCode() {
         return getIdentifier();
     }
 
+    @Override
+    public Collection<Integer> getGenomes() {
+        return Arrays.stream(this.getGraph().getGenomes(this.getIdentifier())).boxed().collect(Collectors.toSet());
```

Comment:
This method is already in `DrawableNode`, so you should make that one `final` and remove this one.